airbyte-cdk/python/airbyte_cdk/sources/declarative/datetime/datetime_parser.py

```diff
@@ -27,6 +27,8 @@ def parse(self, date: Union[str, int], format: str) -> datetime.datetime:
         # See https://stackoverflow.com/a/4974930
         if format == "%s":
             return datetime.datetime.fromtimestamp(int(date), tz=datetime.timezone.utc)
+        elif format == "%float_s":
```

Comment:
Let's call this `%s_as_float`

Reply:
Added documentation and updated the name